triage/render.js

```diff
@@ -128,66 +172,75 @@ function renderCluster(top, key, keyId, text, tests) {
     return count == 1 ? count + ' ' + word : count + ' ' + word + suffix;
   }
 
+  var counts = clustered.makeCounts(keyId);
+
   var clusterSum = clustersSum(tests);
   var recentCount = clustered.getHitsInLastDayById(keyId);
   var failureNode = addElement(top, 'div', {id: keyId}, [
-    createElement('h2',
-      {innerHTML: `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING <a href="#${keyId}" class="key">${keyId}</a>`}),
+    createElement('h2', null, [
+      `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING `,
+      createElement('a', {href: '#' + keyId}, keyId),
+      createElement('a', {href: 'https://github.com/search?type=Issues&q=' + keyId, target: '_blank', rel: 'noopener'}, 'github search')
```

Comment:
Amazing! :clap: 

Would also love to see something I can click to file and issue that prepopulates the keyId in the body for me